test/runtime/Policies.go

```diff
@@ -716,15 +712,15 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 		_, err = vm.PolicyRenderAndImport(script)
 		Expect(err).To(BeNil(), "Unable to import policy: %s", err)
 
-		By(fmt.Sprintf("Pinging host IPv4 (%s) from %s (should work) because it is contained within CIDR %s", ipv4Host, helpers.Httpd2, ipv4OtherHost))
-		res = vm.ContainerExec(helpers.Httpd2, helpers.Ping(ipv4Host))
-		res.ExpectSuccess("Unexpected failure pinging host (%s) from %s: %s", ipv4Host, helpers.Httpd2, res.CombineOutput().String())
+		By(fmt.Sprintf("Pinging host IPv4 (%s) from %s (should work) because it is contained within CIDR %s", helpers.IPv4Host, helpers.Httpd2, ipv4OtherHost))
```

Comment:
For me this By is a bit redundant, the assert message already explain the context of the test no?